<launch>
    <arg name="max_window_size" default="1" />
    <arg name="slope" default="1.0" />
    <arg name="initial_distance" default="0.05" />
    <arg name="max_distance" default="2.0" />
    <arg name="input_topic" default="/cloud_all_fields_fullframe" />
    <arg name="non_ground_topic" default="/non_ground_points" />
    <arg name="ground_topic" default="/ground_points" />

    <node pkg="pmf_ground_filter" type="pmf_ground_filter" name="pmf_ground_filter" output="screen">
        <param name="max_window_size" value="$(arg max_window_size)" />
        <param name="slope" value="$(arg slope)" />
        <param name="initial_distance" value="$(arg initial_distance)" />
        <param name="max_distance" value="$(arg max_distance)" />
        <param name="input_topic" value="$(arg input_topic)" />
        <param name="non_ground_topic" value="$(arg non_ground_topic)" />
        <param name="ground_topic" value="$(arg ground_topic)" />
    </node>
</launch>
